ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -6,8 +6,8 @@
 import SwiftUI
 import UIKit
 
-/// Properties available to customize the state of the Activity Indicator state
-@objc public protocol MSFActivityIndicatorState {
+/// Properties available to customize the configuration of the Activity Indicator configuration
+@objc public protocol MSFActivityIndicatorConfiguration {
```

Comment:
Beyond the discussion behind the naming of these classes and objects, we need be aware that this will cause an extra disruption on the integration of the vnext-tokens branch to main which will likely make it an even longer process.

In the devmain environment this will mean a coordinated update of all indirect dependencies along with the code changes in devmain itself.

We can talk about this offline too, but I believe this change could go in a separate smaller and focused integration to prevent multiple potential issues in only a single big integration.

Reply:
The merge to `main`, ignoring devmain, won't be disrupted by this change, since we've done a good job of FIing from `main` into `vnext-*`.

Your point about devmain, however, is fair! That said, we may yet need to coordinate downstream dependencies since there have been a few minor public API changes along with this. We won't know for sure until we test, however.

If we did break this into a separate integration, presumably it would have to wait for another minor version change, right? So tokens are 0.5.x, and then this change would create 0.6.x?